tensorflow_lite_support/python/task/audio/audio_classifier.py

```diff
@@ -19,15 +19,15 @@
 from tensorflow_lite_support.python.task.audio.core import tensor_audio
 from tensorflow_lite_support.python.task.audio.core.pybinds import _pywrap_audio_buffer
 from tensorflow_lite_support.python.task.audio.pybinds import _pywrap_audio_classifier
-from tensorflow_lite_support.python.task.core.proto import base_options_pb2
+from tensorflow_lite_support.python.task.core import base_options
```

Comment:
Could you rename this to `from tensorflow_lite_support.python.task.core import base_options as base_options_module` because in the code below, there are places where a variable named `base_options` are defined, such as on line 65, creating a lot of lint error:
```
Redefining name 'base_options' from outer scope (line 22) 
```

Please apply this change to all files in this PR.

Reply:
Yeah I can see why it would clash. I’ll update it soon.

Reply:
@khanhlvg I've updated them to use the new import.